lib/src/main/java/com/soundcloud/android/crop/CropImageActivity.java

```diff
@@ -218,24 +220,28 @@ private void makeDefault() {
             final int height = rotateBitmap.getHeight();
 
             Rect imageRect = new Rect(0, 0, width, height);
-
-            // Make the default size about 4/5 of the width or height
-            int cropWidth = Math.min(width, height) * 4 / 5;
-            @SuppressWarnings("SuspiciousNameCombination")
-            int cropHeight = cropWidth;
-
-            if (aspectX != 0 && aspectY != 0) {
+            RectF cropRect = null;
+            if (!defaultNoCrop) {
+              // Make the default size about 4/5 of the width or height
+              int cropWidth = Math.min(width, height) * 4 / 5;
```

Comment:
good points. I've since forked my own copy of the library.

On Fri, Apr 17, 2015 at 3:56 PM, Jamie McDonald <notifications@github.com>
wrote:

> In lib/src/main/java/com/soundcloud/android/crop/CropImageActivity.java
> <https://github.com/jdamcd/android-crop/pull/71#discussion_r28625129>:
>
> > @@ -218,24 +220,28 @@ private void makeDefault() {
> >              final int height = rotateBitmap.getHeight();
> >
> >              Rect imageRect = new Rect(0, 0, width, height);
> > -
> > -            // Make the default size about 4/5 of the width or height
> > -            int cropWidth = Math.min(width, height) * 4 / 5;
> > -            @SuppressWarnings("SuspiciousNameCombination")
> > -            int cropHeight = cropWidth;
> > -
> > -            if (aspectX != 0 && aspectY != 0) {
> > +            RectF cropRect = null;
> > +            if (!defaultNoCrop) {
> > +              // Make the default size about 4/5 of the width or height
> > +              int cropWidth = Math.min(width, height) * 4 / 5;
>
> 4 space tabs for consistency with the rest of the project
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdamcd/android-crop/pull/71/files#r28625129>.
>